classes/submission/maps/Schema.php

```diff
@@ -526,16 +526,33 @@ protected function canChangeMetadata(Enumerable $stageAssignments, Submission $s
     /**
      * Get details about the review assignments for a submission
      */
-    protected function getPropertyReviewAssignments(Enumerable $reviewAssignments, bool|Collection $anonymizeReviews = false): array
+    protected function getPropertyReviewAssignments(Enumerable $reviewAssignments, bool|Collection $anonymizeReviews = false, Submission $submission, array $stages): array
     {
-        $reviews = [];
         $request = Application::get()->getRequest();
         $currentUser = $request->getUser();
 
+        // application specific review stages
+        $reviewStageIds = Application::get()->getReviewStages();
+
+        // get roles from review stages only
+        $currentUserAssignedRoles = [];
```

Comment:
Is this variable used anywhere? It looks like we can remove it and the `foreach` block which retrieves it.